cmd/discovery.go

```diff
@@ -22,7 +26,19 @@ var rootCmd = &cobra.Command{
 	Long:  `https://github.com/quantum/discovery`,
 	Run: func(cmd *cobra.Command, args []string) {
 		etcdMachines := strings.Split(etcdURLs, ",")
-		err := handling.InitAndServe(discoveryAddr, etcdMachines)
+		cfg := client.Config{
```

Comment:
suggestion to split out this inline method to keep the cobra command looking simple